solution/backend/regulations/views/homepage.py

```diff
@@ -24,7 +23,7 @@ def get_context_data(self, **kwargs):
         if not parts:
             return context
 
-        full_structure = get_structure(parts)
+        full_structure = Title.objects.all().values_list("toc", flat=True)
```

Comment:
Not a problem I see we are using teh get_structure only in one other location.  Are we essentially just using this instead of what you did for the TOC?  Can we just do the same there with no issue.  Just a code clean up thing.  Wouldnt include it here but Just a small story for next sprint